lti_tool/models.py

```diff
@@ -466,6 +466,35 @@ def __str__(self):
         return self.title if self.title else self.id_on_platform
 
 
+class LtiLineItem(models.Model):
+    """Represents an LTI LineItem for assignment scoring.
+
+    The line item scope and methods are described in the LTI Assignment and Grade
+    Services 2.0 spec at
+    https://www.imsglobal.org/spec/lti-ags/v2p0#line-item-service-scope-and-allowed-http-methods
+    """
+
+    context = models.ForeignKey(
+        LtiContext,
+        related_name="line_items",
+        on_delete=models.CASCADE,
+        verbose_name=_("context"),
+    )
+    url = models.URLField(_("URL on Platform"))
```

Comment:
Should we make this unique, since it also doubles as an identifier?

Reply:
Good call. There should never be two references that share an ID, especially since they're segmented by URL rather than an opaque identifier that might conflict across platforms.

- [ ] Make `url` unique